src/detail-page/components/StyledTab.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
What do you think about adding a new folder inside `detail-page/components` directory that would contain out tab components (without tab content components)? :)
![image](https://user-images.githubusercontent.com/8350925/107555812-e8a1f180-6bd7-11eb-8d13-d62a5ea297f9.png)

Reply:
Yes! I is good idea. :)